frameworks/PHP/symfony/src/Controller/FortunesController.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Fortune;
+use App\Repository\FortuneRepository;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Twig\Environment;
+
+class FortunesController
+{
+    /** @var Environment */
+    private $twig;
+    /** @var FortuneRepository */
+    private $fortuneRepository;
+
+    public function __construct(Environment $twig, FortuneRepository $fortuneRepository)
+    {
+        $this->twig = $twig;
+        $this->fortuneRepository = $fortuneRepository;
+    }
+
+    /**
+     * @Route("/fortunes")
+     * @throws
```

Comment:
those seem irrelevant

Reply:
i was reluctant to add them - phpstorm marks half of the controller code as unchecked function calls :|

i should probably remove them